slapo/model_schedule/gpt.py

```diff
@@ -1,15 +1,113 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
+"""HuggingFace GPT-Neo with model schedule."""
```

Comment:
We could also rename gpt to gpt_neo to better differentiate gpt2 (both here and examples/). This could also better connect to the name under transformers/model/gpt_neo.